src/lib/containers/access_requirement_list/managedACTAccess/RequestDataAccessStep2.tsx

```diff
@@ -120,8 +128,28 @@ const RequestDataAccessStep2: React.FC<RequestDataAccessStep2Props> = props => {
     // get data access required docs data to display file names
     getFilesData(dataAccessRequestData)
 
+    // renewal case
     if (dataAccessRequestData.concreteType === 'org.sagebionetworks.repo.model.dataaccess.Renewal') {
       setIsRenewal(true)
+      if (dataAccessRequestData.accessorChanges?.length) {
```

Comment:
This seems generally dangerous.  In the case of a Renewal, could you confirm that these are not already set to RENEW_ACCESS from the backend?  
My concern is when a user saves a request that's in progress.  I think newly added accessors (who should have GAIN_ACCESS), and existing accessors who you've previously changed to REVOKE_ACCESS, will be flipped to RENEW_ACCESS when this UI is reopened.

Reply:
If they initially come back as RENEW_ACCESS, then this code can be safely deleted.

Reply:
I do see RENEW_ACCESS coming from the backend if a data access request has been approved. So I have updated the code to copy the user access types from the backend API to the state variable rather than setting them all to RENEW_ACCESS.